browser-extension/addTournesolRecommendations.css

```diff
@@ -28,25 +28,74 @@
 
 .language_option {
   color: var(--yt-spec-text-primary);
-  font-family: 'Roboto', 'Arial', sans-serif;
+  font-family: "Roboto", "Arial", sans-serif;
   float: right;
   margin-top: 3px;
-  margin-left: 4px;
+  margin-left: 2px;
 }
 
 #tournesol_link {
   color: var(--yt-spec-text-primary);
-  font-family: 'Roboto', 'Arial', sans-serif;
+  font-family: "Roboto", "Arial", sans-serif;
   position: absolute;
   right: 0px;
   top: 15px;
 }
 
+#refresh_button {
```

Comment:
To avoid potential conflicts with other elements we don't control on the page, it would be nice to use a `tournesol_` prefix (already used on previous items) in this id.